namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-staging/resources/elasticache.tf

```diff
@@ -0,0 +1,23 @@
+module "ec-cluster-prison-visits-booking-staff" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.3"
```

Comment:
I think you need to use `github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=3.0` here.

Reply:
Thanks @alkar - will update now.